pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -70,7 +70,7 @@ func (uc *UpgradeCluster) UpgradeCluster(subscriptionID uuid.UUID, resourceGroup
 	}
 
 	switch ucs.OrchestratorProfile.OrchestratorVersion {
```

Comment:
@rjtsdl,

> The current upgrade feature is not working anyway

Where did you get this info? Can you provide more details. Upgrade works today and please make sure that this PR doesn't break it.